cocinaborcelle/src/pages/api/cardapio/[slug].ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import prisma from "../../../../prisma/index"
+
+
+async function showCardapio(req: NextApiRequest, res: NextApiResponse) {
+    if (req.query.slug === "index" && req.method === "GET") {
+        try {
+            const cardapio = await prisma.cardapio.findMany( {
```

Comment:
```suggestion
            const cardapio = await prisma.cardapio.findUnique( {
                where: {id: 1},
```

utilizando findMany, o prisma retorna um array de cardapios, então o json que tava retornando da request ficava dentro de um array. Porém, nós queremos apenas o único cardápio do banco de dados e não receber um array no json de retorno. Para isso, sugiro utilizar o método .findUnique e passar o id do único cardápio do banco de dados (id: 1)